src/SFA.DAS.Events.Application/Queries/GetApprenticeshipEvents/GetApprenticeshipEventsQueryHandler.cs

```diff
@@ -28,10 +28,10 @@ public GetApprenticeshipEventsQueryHandler(IApprenticeshipEventRepository appren
 
         private static IEnumerable<ApprenticeshipEvent> MapFrom(IEnumerable<ApprenticeshipEvent> source)
```

Comment:
for mapping entities to api types (there's a separate library for api client types)